README.md

```diff
@@ -80,15 +80,23 @@ To deploy to staging or production, you need to be a core member of the Open Col
 
 ### Staging (heroku)
 
-```
+```bash
+# Before first deployment, configure staging remote
+git remote add staging https://git.heroku.com/oc-staging-frontend.git
+
+# Then deploy master with
```

Comment:
```suggestion
# Then deploy staging with
```

Reply:
It was in the sense that we deploy `master` branch. Maybe I can be clearer by saying `Then deploy master branch to staging with`?